### DISCUSSION TITLE
Salt-minion Errors

### DISCUSSION THREAD

**Discussion by cvfdblaze624** [Security-Onion-Solutions/securityonion#5349] [Unsupported Versions]
Hello all, having a minor issue I can't seem to pin down.  

SO is working, so-status all green, but I am getting blasted with salt-minion errors as shown below.  Currently have a hardware install, running newest version 2.3.70, installed from git after install of Ubuntu 18.04.  The issue appeared when I did a SOUP from  2.3.30 to 2.3.60.  Everything else is working fine, and other than the errors in the logs it doesn't SEEM to be causing issues.  I ran a salt-call -l debug state.apply and the only errors were the ones below.  

I am wondering since I waited for a few updates before doing a SOUP if something didn't update correctly.  Thanks for any suggestions!!

[ERROR   ] Command '/usr/bin/patch' failed with return code: 1
[ERROR   ] stdout: patching file /tmp/__salt.tmp.0h2aj5p5 (read from /usr/lib/python3/dist-packages/salt/states/influxdb_continuous_query.py)
[ERROR   ] retcode: 1
[ERROR   ] Command '/usr/bin/patch' failed with return code: 1
[ERROR   ] stdout: patching file /tmp/__salt.tmp.p2kjl2q_ (read from /usr/lib/python3/dist-packages/salt/states/influxdb_retention_policy.py)
[ERROR   ] retcode: 1
[ERROR   ] Command '/usr/bin/patch' failed with return code: 1
[ERROR   ] stdout: patching file /tmp/__salt.tmp.oto5czyt (read from /usr/lib/python3/dist-packages/salt/modules/influxdbmod.py)
[ERROR   ] retcode: 1

**Comment by m0duspwnens** [comment]
@cvfdblaze624 
 Can you check the files `/usr/lib/python3/dist-packages/salt/states/influxdb_continuous_query.py`, `/usr/lib/python3/dist-packages/salt/states/influxdb_retention_policy.py`, and `/usr/lib/python3/dist-packages/salt/modules/influxdbmod.py` and verify that that they have the appropriate patches applied? Each file should have the patch applied from the patch file matching its name located [here](https://github.com/Security-Onion-Solutions/securityonion/tree/master/salt/salt/files).

**Reply by cvfdblaze624** [reply]
I checked and if I am reading the code correctly and assuming the green text on those files is what should be after patching, it appears the patch has been performed.  I also noticed the requires python-influxdb statement, it wasn't installed so I added it to test and still same error on salt-call appears
